lib/primitives/object-element.js

```diff
@@ -45,6 +45,14 @@ module.exports = function(BaseElement, ArrayElement, MemberElement) {
       );
     },
 
+    delete: function (name) {
+      this.content = this.content.filter(function (item) {
+        return item.key.toValue() !== name;
+      });
+
+      return this;
```

Comment:
Why would it return `this`? Why not return the deleted element like the other collection removal APIs in JS.